pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/cluster.go

```diff
@@ -11,8 +11,8 @@ package k8s
 import (
 	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/collectors"
 	k8sProcessors "github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/processors/k8s"
-	"github.com/DataDog/datadog-agent/pkg/orchestrator"
 
+	model "github.com/DataDog/agent-payload/v5/process"
```

Comment:
nit: can we use the same package name in all files (agentModel vs model)

Reply:
[unify import pkg name](https://github.com/DataDog/datadog-agent/pull/15136/commits/2f2579515d8d17a8773510c285514c860cee1569)